src/components/accordion/Accordion.js

```diff
@@ -84,14 +84,19 @@ export class LeuAccordion extends LeuElement {
         <span class="label">${this.label}</span>
         <div class="plus"></div>
       </button></${unsafeStatic(hTag)}>
-      <div
-        id="content"
-        class="content"
-        aria-labelledby="toggle"
-        role="region"
+      <div 
+        id="contentwrapper"
+        class="contentwrapper"  
         ?hidden=${!this.open}
       >
-        <slot name="content"></slot>
+        <div
+          id="content"
```

Comment:
This id can be removed after the `aria-controls` attribute of the toggle is updated.

Reply:
done